types/port.pp

```diff
@@ -0,0 +1,13 @@
+# @summary ferm port-spec
```

Comment:
Could you please add a spec test for this new type. Here's a simple example

https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/spec/type_aliases/yes_no_spec.rb

You end up creating two arrays, one of things that should match and one of things that should not.

Reply:
Will do, thanks for the tip!

Reply:
@ghoneycutt I've amended a test for `Ferm::Port` and added tests for the other ferm types in a separate commit as well, please have a look if that is what you had in mind.